pyproject.toml

```
@@ -285,7 +285,8 @@ devel-devscripts = [
     "twine>=4.0.2",
 ]
 devel-duckdb = [
-    "duckdb>=0.9.0",
+    # TODO: duckdb dev wheels are produced in a strange way with 3.12 support (they are missing for linux now)
+    "duckdb>=0.9.0; python_version < '3.12'",
```

https://github.com/duckdb/duckdb/releases/tag/v0.10.0

```suggestion
    "duckdb>=0.10.0",
```

---

Or selective per version

```suggestion
    "duckdb>=0.9.0; python_version < '3.12'",
    "duckdb>=0.10.0; python_version == '3.12'",
```

---

Cool. Updated.